backend/lcfs/web/api/user/schema.py

```diff
@@ -28,12 +27,11 @@ class UserCreate(BaseModel):
     last_name: str
     is_active: bool
     organization_id: Optional[int] = None
-    organization: OrganizationSummarySchema
-    roles: List[RoleSchema]
+    organization: OrganizationSummarySchema = Field(exclude=True)
```

Comment:
What's this used for 'Field(exclude=True)'?

Reply:
As it's for create, trying to exclude the organization and roles getting loaded from model.